### DISCUSSION TITLE
做了个高适配Xray的LuCI，需要技术指点

### DISCUSSION THREAD

**Discussion by BI7PRK** [XTLS/Xray-core#1651] [General]
## 初衷
因为在OpenWRT下并没有针对Xray的LuCI，而V2Ray则已有比较成熟的LuCI，在过去两者高度兼容，LuCI可通用。如今似乎越来越有差异化，从而提取了一个 `luci-app-v2ray` 代码，通过修改适配XRay并取名为 `luci-app-xray`。[项目代码见](https://github.com/BI7PRK/luci-app-xray)

自动化编译出错。环境需要安装 >=python3.5。对于github 的workflow不熟悉（其实是不懂）。在代码中加入了 `python3` 也不行。
```SHELL
 - name: Install Dependencies
      run: |
        DEBIAN_FRONTEND=noninteractive \
        sudo apt-get install -y ccache gettext libncurses5-dev xsltproc gcc g++ python3
```

## 请指教
如何在环境中安装 >=python3.5

**Comment by 3xpert** [comment]
https://github.com/yichya/luci-app-xray
